frontend/src/app/patients/personSchema.ts

```diff
@@ -84,7 +84,16 @@ export function areValidPhoneNumbers(phoneNumbers: any) {
     return phoneNumberIsValid(phoneNumber.number);
   });
 }
-
+const isValidDate = (date: any) => {
```

Comment:
can we explicitly type `date` as a `string` here? Or can it possibly be something else?

Reply:
Resolved in 483d8827